rollup.config.js

```diff
@@ -29,6 +29,37 @@ const umd_config = {
   ],
 };
 
+const cli_config = {
```

Comment:
[The rollup documentation](https://rollupjs.org/guide/en/#config-intellisense) suggests using or JSDoc comment, or `defineConfig` helper to leverage IDE's Intellisense